cfpb/cfgov-refresh
# Adds media image size modifiers

Adds media sizing classes to the media blocks.

## Additions

- Added `media_image__100` and `media_image__130-to-150` classes.

## Testing

- Add `media_image__100` to an `media_image` element and observe the size set to 100x100px across all screen sizes.
- Add `media_image__130-to-150` to an `media_image` element and observe the size set to 130x130px up to 800px wide, whereby it changes to 150x150px.

## Review

- @jimmynotjim 
- @KimberlyMunoz 
- @sebworks 

## Notes

- Spun off from https://github.com/cfpb/cfgov-refresh/pull/686 as part of DF-2099. These are used to set the image sizes in the media images used in the carousels on the careers and the bureau pages.
